### PR TITLE
Support 644 and 640 permissions as well

### PR DESCRIPTION
**Pull request by gugu**

### CODE REVIEW

**Comment by MichaelAquilina** [comment]
Could you add a test for this similar to this one: https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv/blob/master/tests/test_check_venv.zunit#L28-L42

Once that's been add it - I think we can go ahead and merge this change :)

**Comment by gugu** [comment]
Done

File: tests/test_check_venv.zunit [hunk 1]
```diff
@@ -41,6 +41,67 @@
...
+    MYOLDPWD="$(dirname $TARGET)"
+    AUTOSWITCH_DEFAULTENV="foodefault"
+
+    run check_venv
+
+    assert $status equals 0
+    assert "$output" same_as "Switching virtualenv: foobar  [$PYTHON_VERSION]"
+}
+
+@test 'check_venv - No security warning for readable by group permission' {
```
Comment (MichaelAquilina) [inline tests/test_check_venv.zunit]: this looks like a duplicate

**Comment by MichaelAquilina** [comment]
This fix is now included in 0.3.6

**Pull request merged**